packages/ui/src/components/KafkaInstanceDrawer/components/KafkaSampleCode.tsx

```diff
@@ -0,0 +1,262 @@
+            isSelected={clientSelect === "quarkus"}
+            onChange={() => setClientSelect("quarkus")}
+          />
+          <ToggleGroupItem
+            text={t("sample_code.clients.spring_boot")}
+            value="springboot"
+            buttonId="springboot"
+            isSelected={clientSelect === "springboot"}
+            onChange={() => setClientSelect("springboot")}
+          />
+        </ToggleGroup>
+        <Text component={TextVariants.h3} className="pf-u-mt-xl">
+          {t("sample_code.sample_configuration_code")}
+        </Text>
+        <Text component={TextVariants.small}>
+          {t("sample_code.sample_configuration_code_description")}
+        </Text>
+        <Text component={TextVariants.small}>
+          {t("sample_code.bracket_text")}
+        </Text>
+        {(() => {
+          switch (clientSelect) {
+            case "java":
+              return (
+                <SampleCodeSnippet
+                  copyCode={javaConfig}
+                  codeBlockCode={javaConfig.substring(0, 300)}
+                  expandableCode={javaConfig.substring(300)}
+                />
+              );
+            case "python":
+              return (
+                <SampleCodeSnippet
+                  copyCode={pythonConfig}
+                  codeBlockCode={pythonConfig.substring(0, 300)}
+                  expandableCode={pythonConfig.substring(300)}
+                />
+              );
+            case "quarkus":
+              return (
+                <SampleCodeSnippet
+                  copyCode={quarkusConfig}
+                  codeBlockCode={quarkusConfig.substring(0, 300)}
+                  expandableCode={quarkusConfig.substring(300)}
+                />
+              );
+            case "springboot":
+              return (
+                <SampleCodeSnippet
+                  copyCode={springBootConfig}
+                  codeBlockCode={springBootConfig.substring(0, 300)}
+                  expandableCode={springBootConfig.substring(300)}
+                />
+              );
+            default:
+              return null;
+          }
+        })()}
+        <Text component={TextVariants.h3} className="pf-u-mt-xl">
+          {t("sample_code.sample_producer_code")}
+        </Text>
+        <Text component={TextVariants.small}>
+          <Trans
+            ns={"kafka"}
+            i18nKey={"sample_code.sample_producer_code_description"}
+            values={{
+              client_type: clientSelect,
+            }}
+          />
+        </Text>
+        <Text component={TextVariants.small}>
+          {t("sample_code.bracket_text")}
+        </Text>
+        {(() => {
+          switch (clientSelect) {
+            case "java":
+              return (
+                <SampleCodeSnippet
+                  copyCode={javaProducer}
+                  codeBlockCode={javaProducer.substring(0, 300)}
+                  expandableCode={javaProducer.substring(300)}
```

Comment:
@riccardo-forina the issue that you noticed is happening because I divided the code string into 2parts using substring, I guess I need to follow some other approach